src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotUseObsoleteKDFAlgorithm.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    public class DoNotUseObsoleteKDFAlgorithmTests : DiagnosticAnalyzerTestBase
+    {
+        [Fact]
+        public void TestNormalMethodOfPasswordDeriveBytesDiagnostic()
+        {
+            VerifyCSharp(@"
+using System;
+using System.Security.Cryptography;
+
+class TestClass
+{
+    public void TestMethod(string password, byte[] salt)
+    {
+        new PasswordDeriveBytes(password, salt).GetBytes(1);
+    }
+}",
+            GetCSharpResultAt(9, 9, DoNotUseObsoleteKDFAlgorithm.Rule, "PasswordDeriveBytes", "GetBytes"),
+            GetCSharpResultAt(9, 9, DoNotUseObsoleteKDFAlgorithm.Rule, "PasswordDeriveBytes", "PasswordDeriveBytes"));
```

Comment:
I think we should avoid multiple warnings for this case.  Maybe we should only look for PasswordDervieBytes' constructor?